package.json

```diff
@@ -11,15 +11,20 @@
   "scripts": {
     "start": "node app.js",
     "pretest": "npm run lint",
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:e2e": "echo 'no e2e tests available.'",
+    "test": "npm run test:unit",
+    "test:e2e": "npm run selenium:install && npm run selenium:start && npm run nightwatch",
```

Comment:
Yeah, you can't run just the nightwatch job on it's own.  It will fail.  I'm not sure if there's a way to prevent this.  You have to run test:e2e and that will do everything